src/webapp/reports/glass-data-submission/glass-data-submission-list/useDataSubmissionList.tsx

```diff
@@ -27,12 +29,21 @@ export function useDataSubmissionList(filters: Filter) {
         filters
     );
 
+    const questionnaireDataSets = useMemo(() => {
```

Comment:
this is not being used anywhere?

Reply:
i'm using this to filter for which columns to show in the table. if there are no questionnaires for the selected module, the Questionnaire Completed column is not included in the table

Reply:
ah yes, i see it now.